client/src/context/types.js

```diff
@@ -14,3 +14,9 @@ export const SET_LOADING='SET_LOADING'
 // USER CONTEXT
 export const GET_SELECTED_USER='GET_SELECTED_USER'
 export const CLEAR_SELECTED_USER='CLEAR_SELECTED_USER'
+
+// POST CONTEXT
```

Comment:
thanks for the comments separating out the types. you can also define the type 

`export const ACCESS_TOKEN_FETCH_INFLIGHT = 'ACCESS_TOKEN_FETCH_INFLIGHT';`
`export type ACCESS_TOKEN_FETCH_INFLIGHT = typeof ACCESS_TOKEN_FETCH_INFLIGHT;`